eloquentjs/4_1_sum_of_a_range.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Returns the range of numbers from between start-end by incrementing by the value of step
+ * If only two parameters are provided the function defaults to a step size of positive 1
+ * @param {*} a 
```

Comment:
You should leave these out if you're not going to elaborate and keep them up to date

Reply:
Correcting the incorrect parameter names for the next PR. To clarify, should I be more detailed with the function description? Unsure how much detail is too much.